code/rounds/InProgressRound.cs

```diff
@@ -210,5 +198,15 @@ public override void OnSecond()
                 }
             }
         }
+
+        public void ExecuteOnWinCondition()
+        {
+            TTTTeam result = IsRoundOver();
+
+            if (result != null)
+            {
+                LoadPostRound(result);
+            }
+        }
```

Comment:
We don't need this, check out `OnSecond()` inside of `InProgressRound()`. Technically, we should probably move to an event-based system where we only check if a player dies, leaves, etc. We can make a ticket for this.

Reply:
I wasn't able to get that to automatically trigger, and I agree the event system sounds better I just failed at being able to make events/hooks.

Reply:
That seems a bit odd, we check every single second if the alive players have changed and if the round is over.

Reply:
I am assuming it's the IsRoundOver check, I will debug it in a bit, and see what it's returning. It's defiantly not the min player check.

Reply:
I debugged it and found that if I revert to only calling OnKilled after changing to spectator inside of ToggledForcedSpectator it will check correctly. I have replaced it with OnKilled call. Now it will call with normal OnSecond

Reply:
Fixed my issue, and kept method for future events that could end the round.